nestkernel/conn_parameter.h

```diff
@@ -83,7 +83,7 @@ class ConnParameter
     return 0;
   }
 
-  static ConnParameter* create( const Token& );
+  static ConnParameter* create( const Token&, const size_t );
```

Comment:
Can you please write documentation for this method. It should also explain the new parameter.